File: freshman year/CSE12/hw6/Stack.c
#include <iostream>
#include "Base.h"
#include "Stack.h"
using namespace std;

/* DO NOT CHANGE:  This file is used in evaluation */

Base * Stack :: Pop (void) {
	return Remove (END);
}

Base * Stack :: Push (Base * element) {
	return Insert (element, END);
}

Base * Stack :: Top (void) {
	return View (END);
}

File: freshman year/CSE12/hw6/Stack.java
/* DO NOT CHANGE:  This file is used in evaluation */

public class Stack extends List {
        
        public Stack(String name) {
                super(name);
        }

        public Base pop () {
                return remove (END);
        }

        public Base push (Base element) {
                return insert (element, END);
        }

        public Base top () {
                return view (END);
        }
}
